frontend/src/metabase/admin/people/components/GroupsListing.jsx

```diff
@@ -165,7 +165,7 @@ function EditingGroupRow({
       <td />
       <td className="text-right">
         <span
-          className="link no-decoration cursor-pointer"
+          className={cx(CS.link, CS.noDecoration, "cursor-pointer")}
```

Comment:
`.link` seems to already include `text-decoration: none`, so I think it should be possible to drop `.noDecoration` here. What do you think?

https://github.com/metabase/metabase/blob/8b0bd4995d974e2a56fbabab2ea0cc3d3e0e54e2/frontend/src/metabase/css/core/link.module.css#L5-L9

```suggestion
          className={cx(CS.link, "cursor-pointer")}
```

Reply:
sounds good to me 😄